crates/database/src/orders.rs

```diff
@@ -600,9 +603,11 @@ pub fn limit_orders_with_outdated_fees(
         ORDERS_FROM,
         " WHERE o.valid_to >= $1 ",
         "AND o.class = 'limit' ",
-        "AND o.surplus_fee_timestamp < $2 ",
+        "AND COALESCE(o.surplus_fee_timestamp, 'epoch') < $2 ",
+        "ORDER BY o.surplus_fee_timestamp ASC NULLS FIRST",
```

Comment:
I'm not 100% up to date on this but do we need `surplus_fee_timestamp` to be nullable. If it's not nullable we could create a btree index for `surplus_fee_timestamp` to speed up finding the most outdated fees.

Reply:
It is currently nullable. We could make it not nullable by using a placeholder timestamp like epoch to indicate there has never been an update. This doesn't matter for btree indexes though because they work fine with null values too. However, an index on surplus_fee does not help here because we also need to determine that the order is still valid (valid_to > now). We have an index on that and after applying the first index, the second one is no longer useful.